client/assets/translations/de.po

```diff
@@ -3,503 +3,834 @@ msgstr ""
 msgstr "Es kann keine Verbindung zum Lobby-Server hergestellt werden."
 
-#: logic/ui/lobby_list.gd
 msgid "DEBUG_ACCOUNT"
 msgstr ""
 
-#: logic/ui/lobby_selection_element.tscn
-msgid "Lobby Name"
-msgstr ""
-
-#: logic/ui/lobby_selection_element.tscn
 msgid "MP_JOIN"
 msgstr "Beitreten"
 
-#: logic/ui/lobby_user_element.tscn
-msgid "User Name"
-msgstr ""
-
-#: logic/ui/lobby_user_element.tscn
 msgid "MP_KICK"
 msgstr "Rauswerfen"
 
-#: logic/ui/map_editor.gd
 msgid "MESSAGE_TITLE_MAP_EDITOR_SAVE_NO_NAME"
 msgstr ""
 
-#: logic/ui/map_editor.gd
 msgid "MESSAGE_TEXT_MAP_EDITOR_SAVE_NO_NAME"
 msgstr ""
 
-#: logic/ui/map_editor_ui.gd logic/ui/map_editor_ui.tscn
 msgid "NEUTRAL"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
-msgid "Example 1"
-msgstr ""
-
-#: logic/ui/map_editor_ui.tscn
-msgid "Example 2"
-msgstr ""
-
-#: logic/ui/map_editor_ui.tscn
-msgid "Example 3"
-msgstr ""
-
-#: logic/ui/map_editor_ui.tscn
 msgid "MAP_SETTINGS"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "NAME"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "AUTHOR"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "PLAYER_SETTINGS"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "PLAYERS"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "MONEY"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "TERRAIN_SETTINGS"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "SELECTED"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "OWNER"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "UNIT_SETTINGS"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "HEALTH"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "FUEL"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "AMMO"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "RESIZE"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "SAVE"
 msgstr "Speichern"
 
-#: logic/ui/map_editor_ui.tscn
 msgid "LOAD"
 msgstr "Laden"
 
-#: logic/ui/map_editor_ui.tscn
 msgid "Resize map"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "HORIZONTAL"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "VERTICAL"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "ACCEPT"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "CANCEL"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
 msgid "Open a File"
 msgstr ""
 
-#: logic/ui/map_editor_ui.tscn
-msgid "*.crm"
-msgstr ""
-
-#: logic/ui/map_selection_element.tscn
 msgid "Map Name"
 msgstr ""
 
-#: logic/ui/map_selection_element.tscn
 msgid "SELECT"
 msgstr "Auswählen"
 
-#: logic/ui/menu.tscn
 msgid "Main Menu"
 msgstr ""
 
-#: logic/ui/settings_content.tscn
 msgid "AUDIO"
 msgstr "Audio"
 
-#: logic/ui/settings_content.tscn
 msgid "MUSIC"
 msgstr "Musik"
 
-#: logic/ui/settings_content.tscn
 msgid "SOUND"
 msgstr "Ton"
 
-#: logic/ui/shop.tscn
 msgid "CLOSE"
 msgstr "Schliessen"
 
-#: logic/ui/shop_element.tscn
-msgid "Anti-Air"
+msgid "BUY"
+msgstr "Kaufen"
+
+#, fuzzy
+#| msgid "ANTI_ALIASING"
+msgctxt "Unit name"
+msgid "ANTI_AIR"
+msgstr "Kantenglättung"
```

Comment:
I think this should be FLAK in AW Reboot camp was FLAK, and I saw a video that they were using FLAK also...